src/modules/infobox/components/FeatureDetail.vue

```diff
@@ -0,0 +1,48 @@
+<script setup>
+import DOMPurify from 'dompurify'
+import { computed, toRefs } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+import SelectableFeature from '@/api/features/SelectableFeature.class.js'
+
+const props = defineProps({
+    feature: {
+        type: SelectableFeature,
+        required: true,
+    },
+})
+
+const { feature } = toRefs(props)
+
+const i18n = useI18n()
+
+const hasFeatureStringData = computed(() => typeof feature.value?.data === 'string')
+
+const featureDataEntries = computed(() => {
+    if (hasFeatureStringData.value) {
+        return []
+    }
+    return Object.entries(feature.value.data).filter(([_, value]) => value)
+})
+function sanitizeHtml(htmlText) {
+    return DOMPurify.sanitize(htmlText)
+}
+</script>
+
+<template>
+    <!-- eslint-disable-next-line vue/no-v-html-->
+    <div v-if="hasFeatureStringData" v-html="sanitizeHtml(feature.data)" />
+    <div v-else class="htmlpopup-container">
+        <div class="htmlpopup-header">{{ feature.layer.name }}</div>
+        <div class="htmlpopup-content">
+            <div v-for="[key, value] in featureDataEntries" :key="key" class="d-flex gap-1">
+                <div class="htmlpopup-content-title">{{ i18n.t(key) }} :</div>
+                <!-- eslint-disable-next-line vue/no-v-html-->
+                <div class="htmlpopup-content-data" v-html="sanitizeHtml(value)"></div>
```

Comment:
The div here should have some CSS tweaking, see result below 
![image](https://github.com/geoadmin/web-mapviewer/assets/67745584/05aaa20f-3cd5-4e74-ad09-ccbcd2f5561d)
You can use the kml from this link for testing again https://sys-s.dev.bgdi.ch/jlxh371ioze1